stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -72,6 +185,127 @@ public View getViewForHighlighting(Object element) {
     return (View)element;
   }
 
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+    for (Method method : getViewGetters()) {
+      boolean caught = false;
+      try {
+        @Nullable ViewDebug.ExportedProperty annotation = getExportedAnnotation(method);
+        final Object returnedValue = method.invoke(element);
+        final String cssName = convertGetterNameToCssName(method.getName());
+
+        if (method.getName().equals("getId")) {
+          @Nullable String id = getIdAttribute(element);
+
+          if (id == null) {
+            styles.store("id", "(none)", false);
+          } else {
+            styles.store("id", id, false);
+          }
+        } else {
+          if (method.getReturnType() == int.class) {
+            int intValue = (int) returnedValue;
+
+            if (canIntBeMappedToString(annotation)) {
+              styles.store(
+                  cssName,
+                  intValue +
+                    "(" + mapIntToStringUsingAnnotations(intValue, annotation) + ")",
+                  false);
+            } else {
+              styles.store(cssName, String.valueOf(intValue), isDefaultValue(intValue, annotation));
+            }
+          } else if (method.getReturnType() == float.class) {
+            float floatValue = (float) returnedValue;
+            styles.store(cssName,
+                String.valueOf(floatValue),
+                isDefaultValue(floatValue,
+                annotation));
+          }
+        }
+      } catch (IllegalAccessException e) {
+        caught = true;
+      } catch (InvocationTargetException e) {
+        caught = true;
+      } finally {
+        if (caught) {
+          // something went terribly wrong... none of the called
+          // methods should throw
+          LogUtil.w("(THIS SHOULD NOT HAPPEN) failed to get the style of element id=" +
```

Comment:
we probably shouldn't be spewing out things like `THIS SHOULD NOT HAPPEN` ... if that's really the case then just use `LogUtil.e` :)